### PR TITLE
Add travis ci support.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
PTAL

**Comment by ngaut** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,17 @@ test:
...
+	go get github.com/golang/lint/golint
+	
 	go tool vet . 
 	go tool vet --shadow . 
 	golint ./... 
-	gofmt -s -l .
\ No newline at end of file
+	gofmt -s -l .
+
+deps:
```
Comment (siddontang) [inline Makefile]: I think we can use `go get -d` directly.

Comment (qiuyesuifeng) [inline Makefile]: A good suggestion.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**